Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "masternodes-diff-processor"
+version = "0.1.0"
+edition = "2018"
+build = "build.rs"
+publish = false
+
+# Please don't forget to add relevant features to docs.rs below
+[features]
+default = [ "std", "secp-recovery" ]
+#base64 = [ "base64-compat" ]
+unstable = []
+rand = ["secp256k1/rand-std"]
+#use-serde = ["serde", "bitcoin_hashes/serde", "secp256k1/serde"]
+secp-lowmemory = ["secp256k1/lowmemory"]
+secp-recovery = ["secp256k1/recovery"]
+
+# At least one of std, no-std must be enabled.
+#
+# The no-std feature doesn't disable std - you need to turn off the std feature for that by disabling default.
+# Instead no-std enables additional features required for this crate to be usable without std.
+# As a result, both can be enabled without conflict.
+std = ["secp256k1/std", "bitcoin_hashes/std", "bech32/std"]
+no-std = ["hashbrown", "core2/alloc", "bitcoin_hashes/alloc"]
+
+[dependencies]
+byte = "0.2"
+bls = "0.0.0"
+sha2 = "0.9.8"
+bech32 = { version = "0.8.1", default-features = false }
+bitcoin_hashes = { version = "0.10.0", default-features = false }
+bitcoinconsensus = { version = "0.19.0-3", optional = true }
+hashbrown = { version = "0.8", optional = true }
+secrets = "1.1.0"
+blake3 = "1.1.0"
+bls-signatures = "0.11.1"
```

Comment:
Which version of BLS is this?  I suppose it is not compatible with our version.

Reply:
Yeah, forgot to clean it up after research.